src/client/src/ui/status-connection/StatusButton.tsx

```diff
@@ -0,0 +1,108 @@
+import _ from 'lodash'
+import React, { Component, SFC, SyntheticEvent } from 'react'
+
```

Comment:
There is too much going on in this file. I would suggest you move Service to another filder, getApplicationStatus to another file as well? StatusButton should be the only coomponent seen here due to Single Responsibility Principle.

Reply:
Inclined to agree with above comment, there is no harm in service moving into its own component file.  Possibly though the getApplicationStatus method is a concern of this component only.